components/dashboard/sections/collectives/AllCollectives.tsx

```diff
@@ -174,7 +174,7 @@ const AllCollectives = ({ subpath }: Omit<DashboardSectionProps, 'accountSlug'>)
                 onEdit: handleEdit,
                 host: data?.host,
                 openCollectiveDetails: handleDrawer,
-              } as HostedCollectivesDataTableMeta
+              } as unknown as HostedCollectivesDataTableMeta
```

Comment:
Not sure how complex this type has got, but perhaps we should simplify the `openCollectiveDetails` type requirements instead.